### DISCUSSION TITLE
[touchscreen] Cursor position is not updated on quick taps if mouse buttons are disabled

### DISCUSSION THREAD

**Discussion by Doksha** [ppy/osu#20823] [Q&A]
### Type

Game behaviour

### Bug description

if you tap quickly with mouse buttons disabled, the location of the cursor is not updated, this can apply to players who aim with touch, but tap with their other hand.
This makes jumps basically impossible without mouse buttons enabled, or dragging every jump

### Screenshots or videos

https://user-images.githubusercontent.com/72625258/196606727-b9b4ddae-475e-4cd7-815b-bafa6454ca64.mp4

(I guess obs cant capture the on screen taps, but both before and after toggling I am tapping at similar frequencies, yet as you can see, only long taps or slides/micro movements are registered)

### Version

2022.1008.2-lazer

### Logs

[performance-update.log](https://github.com/ppy/osu/files/9817156/performance-update.log)
[runtime.log](https://github.com/ppy/osu/files/9817157/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9817158/updater.log)
[database.log](https://github.com/ppy/osu/files/9817159/database.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/9817160/legacy-ipc.log)
[network.log](https://github.com/ppy/osu/files/9817161/network.log)
[performance.log](https://github.com/ppy/osu/files/9817162/performance.log)
[performance-draw.log](https://github.com/ppy/osu/files/9817163/performance-draw.log)
[performance-input.log](https://github.com/ppy/osu/files/9817164/performance-input.log)

**Comment by peppy** [comment]
https://github.com/ppy/osu/issues?q=is%3Aissue+is%3Aopen+touchscreen
https://github.com/ppy/osu/search?q=touchscreen&type=discussions

There are still issues with some touch screen setups. Would need to know what hardware you are using and whether you are using raw input / OTD. and whether you're tried togglign those as they will generally give completely different behaviours.

**Reply by Doksha** [reply]
The issue happens with raw input on and off, and I didnt even know it was possible to use OTD with touchscreen. I doubt it is a hardware issue because it works perfectly fine with mouse buttons enabled (it even works if mouse buttons are unbound, which should be the default behavior anyway.)
To be clear, the _system_ is detecting all taps (every time I tap I always see the windows tap circles), but lazer doesnt detect taps that are too quick

**Reply by peppy** [reply]
Again, Please provide more details on what touch screen.

**Reply by Doksha** [reply]
This is basically all I know about it: https://www.amazon.com/gp/product/B07TXP4CSY
